HealthcareSystem/Backend/Controller/NotificationSurveyAndFeedback/FeedbackController.cs

```diff
@@ -15,16 +15,23 @@ public class FeedbackController
    {
 
         private FeedbackService feedbackService = new FeedbackService();
-      public Model.Users.Feedback AddFeedback(Feedback feedback)
-      {
-            return feedbackService.AddFeedback(feedback);
-      }
-      
-      public List<Model.Users.Feedback> ViewFeedbackInformations()
-      {
-            return feedbackService.ViewFeedbackInformations();
-      }
-   
-   
-   }
+        public void NewFeedback(Feedback feedback)
```

Comment:
Is there a specific reason to name this method 'NewFeedback' instead of 'AddFeedback'? As far as I am aware most of the existing code uses 'AddX' as the name, and since this is definitely something that should be consistent, I feel like eventually switching to 'NewX' everywhere is just unnecessary work.